source/process/pre-release.md

```diff
@@ -16,13 +16,17 @@ Once activated you should see additional options in the System Console and the "
 
 #### Setting up Active Directory/LDAP
 
-Instructions for setting up LDAP/AD SSO are available in the [deployment guide.](http://docs.mattermost.com/deployment/sso-ldap.html)
+Instructions for setting up LDAP/AD SSO are available in the [LDAP deployment guide.](http://docs.mattermost.com/deployment/sso-ldap.html)
+
+#### Setting up SAML Single-Sign-On (SSO)
+
+Instructions for setting up SAML SSO are available in the [SAML deployment guide.](http://docs.mattermost.com/deployment/sso-saml.html)
 
 #### Downgrading from Enterprise Edition to Team Edition 
 
 To downgrade from Enterprise Edition to Team Edition: 
 
-1. Disable Active Directory/LDAP SSO.
+1. Disable Active Directory/LDAP SSO and SAML SSO.
```

Comment:
hmmm for this i think they need to make sure that all accounts switch to another type if authentication?